vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -355,6 +355,22 @@ void TextInputShadowNode::registerEvents() {
           }
         });
   }
+
+  control.as<xaml::UIElement>().AddHandler(
```

Comment:
I see there are other usages like this but why do AddHandler instead of calling the event adder method?
control.as<xaml::UIElement>().PointerPressed([=] (...));

Reply:
We've discussed this in the original PR ;)
These events are swallowed by the XAML TextBox, and I .AddHandler() to specify that I want to handle already-handled events.

Reply:
![image](https://user-images.githubusercontent.com/22989529/107735822-79023400-6cb5-11eb-9c0a-ada760fb1087.png)